pkg/ui/workspaces/cluster-ui/src/contexts/timezoneContext.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+import { createContext, useContext } from "react";
+
+export const TimezoneContext = createContext<string>("UTC");
+
+// WithTimezone wraps a class component to provide the
+// context's timezone value as a component prop.
+export function WithTimezone(Component: any) {
```

Comment:
nit:Isn't there a react component type? I personally prefer to avoid `any` wherever possible.